ghost/admin/app/components/members-activity/table-row.hbs

```diff
@@ -36,7 +36,7 @@
                         {{/if}}
                     </span>
                     {{#if event.description}}
-                        <div class="ghost-members-activity-event-description {{if (feature "memberAttribution") 'feature-memberAttribution'}}">
+                        <div class="ghost-members-activity-event-description feature-memberAttribution">
```

Comment:
Removing these classes and updating the associated CSS files have side effects sadly, so I'm currently going to keep these in